regolith/filter_nim.go

```diff
@@ -32,6 +36,16 @@ func NimFilterDefinitionFromObject(
 			jsonPropertyTypeError, "script", "string")
 	}
 	filter.Script = script
+
+	requirementsObj, ok := obj["requirements"]
```

Comment:
I'm not sold on "requirements" as a property name, necessarily. I think I would prefer `requirementsPath`. Although I guess we have `script` not `scriptPath` so... hmm. Maybe it's fine.

Reply:
That's the reason why I chose the "requirements" not "requirementsPath" I also thought about naming it like that.